.eslintrc.json

```diff
@@ -88,7 +88,7 @@
   "rules": {
     "eqeqeq": "error",
     "@typescript-eslint/explicit-function-return-type": "off",
-    "@typescript-eslint/no-unused-vars": ["error", { "argsIgnorePattern": "^_" }],
+    "@typescript-eslint/no-unused-vars": ["error", { "argsIgnorePattern": "^_", "varsIgnorePattern": "^_" }],
```

Comment:
should it be a separate PR just updating the rule ?
also it may be odd to ignore a variable like maybe not declare it ?

Reply:
I finally changed the way I was removing the first element of the array, to avoid changing this rule